### ISSUE TITLE
issue related to error at  x = torch.cat([z,l,k],dim=1)  models.py file

### ISSUE THREAD

**Issue by praveenkumarchandaliya** [Jooong/Face-Aging-CAAE-Pytorch#1]
![img_20180411_161542](https://user-images.githubusercontent.com/27194129/38653137-da78b968-3e26-11e8-8b1d-0c4601627df5.jpg)

x = torch.cat([z,l,k],dim=1)
RuntimeError:Expected objet of type Variable torch.cuda.FloatTensor
type variable  torch cuda.LongTensor at position #2 for iterable argument tensors

I am using Pytorch 0.30

**Comment by wjxdp** [comment]
@praveenkumarchandaliya this error has caused by pytorch version,maybe you can change "x = torch.cat([z,l,k],dim=1)" to "x = torch.cat((z,l,k),dim=1)"

**Comment by Jooong** [comment]
fixed by casting tensor type
